<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2018-01-10T12:08:56Z" name="Default suite" finished-at="2018-01-10T12:10:52Z" duration-ms="115427">
    <groups>
      <group name="sanity-group">
        <method signature="SignUpByFacbookTest.DoLoginWithFacbook()[pri:5, instance:Tests.SignUpByFacbookTest@4803b726]" name="DoLoginWithFacbook" class="Tests.SignUpByFacbookTest"/>
        <method signature="SignUpByFacbookTest.LoginWithFacbookWithoutPassword()[pri:4, instance:Tests.SignUpByFacbookTest@4803b726]" name="LoginWithFacbookWithoutPassword" class="Tests.SignUpByFacbookTest"/>
        <method signature="SignUpByFacbookTest.LoginWithFacbookWithoutUser()[pri:3, instance:Tests.SignUpByFacbookTest@4803b726]" name="LoginWithFacbookWithoutUser" class="Tests.SignUpByFacbookTest"/>
        <method signature="SignUpByFacbookTest.LoginWithFacbookWrongPassword()[pri:2, instance:Tests.SignUpByFacbookTest@4803b726]" name="LoginWithFacbookWrongPassword" class="Tests.SignUpByFacbookTest"/>
        <method signature="SignUpByFacbookTest.LoginWithFacbookWrongUserName()[pri:1, instance:Tests.SignUpByFacbookTest@4803b726]" name="LoginWithFacbookWrongUserName" class="Tests.SignUpByFacbookTest"/>
      </group> <!-- sanity-group -->
    </groups>
    <test started-at="2018-01-10T12:08:56Z" name="Default test" finished-at="2018-01-10T12:10:52Z" duration-ms="115427">
      <class name="Tests.SignUpByFacbookTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:Tests.SignUpByFacbookTest@4803b726]" started-at="2018-01-10T14:08:56Z" name="setup" finished-at="2018-01-10T14:09:28Z" duration-ms="31565" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="LoginWithFacbookWrongUserName()[pri:1, instance:Tests.SignUpByFacbookTest@4803b726]" started-at="2018-01-10T14:09:28Z" name="LoginWithFacbookWrongUserName" finished-at="2018-01-10T14:09:42Z" duration-ms="14408" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithFacbookWrongUserName -->
        <test-method signature="LoginWithFacbookWrongPassword()[pri:2, instance:Tests.SignUpByFacbookTest@4803b726]" started-at="2018-01-10T14:09:42Z" name="LoginWithFacbookWrongPassword" finished-at="2018-01-10T14:09:57Z" duration-ms="14729" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithFacbookWrongPassword -->
        <test-method signature="LoginWithFacbookWithoutUser()[pri:3, instance:Tests.SignUpByFacbookTest@4803b726]" started-at="2018-01-10T14:09:57Z" name="LoginWithFacbookWithoutUser" finished-at="2018-01-10T14:10:03Z" duration-ms="6180" status="FAIL">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[An unknown server-side error occurred while processing the command. Original error: Error while executing atom: Element is no longer attached to the DOM (status: 10) (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 0 millisecondsBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:32.194Z'System info: host: 'yinons-MacBook-Pro.local', ip: 'fe80:0:0:0:1002:5be7:d86e:1f09%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.6', java.version: '9.0.1'Driver info: io.appium.java_client.ios.IOSDriverCapabilities {app: /Users/yinonwishi/Desktop/W..., automationName: XCuiTest, browserName: , databaseEnabled: false, deviceName: iPhone 7, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, newCommandTimeout: 50000, platform: MAC, platformName: MAC, platformVersion: 11.2, startIWDP: true, takesScreenshot: true, udid: FE6347AA-9FAB-48FB-88BC-23F..., webStorageEnabled: false}Session ID: 8db4b8ad-993e-4ba8-a136-3332c530fe54]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Error while executing atom: Element is no longer attached to the DOM (status: 10) (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 0 millisecondsBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:32.194Z'System info: host: 'yinons-MacBook-Pro.local', ip: 'fe80:0:0:0:1002:5be7:d86e:1f09%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.6', java.version: '9.0.1'Driver info: io.appium.java_client.ios.IOSDriverCapabilities {app: /Users/yinonwishi/Desktop/W..., automationName: XCuiTest, browserName: , databaseEnabled: false, deviceName: iPhone 7, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, newCommandTimeout: 50000, platform: MAC, platformName: MAC, platformVersion: 11.2, startIWDP: true, takesScreenshot: true, udid: FE6347AA-9FAB-48FB-88BC-23F..., webStorageEnabled: false}Session ID: 8db4b8ad-993e-4ba8-a136-3332c530fe54at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:89)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:279)at io.appium.java_client.DefaultGenericMobileElement.execute(DefaultGenericMobileElement.java:45)at io.appium.java_client.MobileElement.execute(MobileElement.java:1)at io.appium.java_client.ios.IOSElement.execute(IOSElement.java:1)at org.openqa.selenium.remote.RemoteWebElement.isEnabled(RemoteWebElement.java:150)at PageObjects.BasePage.ElementEnable(BasePage.java:76)at PageObjects.FacbookPage.LoginWithFacbookWithoutUser(FacbookPage.java:74)at Tests.SignUpByFacbookTest.LoginWithFacbookWithoutUser(SignUpByFacbookTest.java:31)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.base/java.lang.reflect.Method.invoke(Method.java:564)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithFacbookWithoutUser -->
        <test-method signature="LoginWithFacbookWithoutPassword()[pri:4, instance:Tests.SignUpByFacbookTest@4803b726]" started-at="2018-01-10T14:10:03Z" name="LoginWithFacbookWithoutPassword" finished-at="2018-01-10T14:10:44Z" duration-ms="41184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithFacbookWithoutPassword -->
        <test-method signature="DoLoginWithFacbook()[pri:5, instance:Tests.SignUpByFacbookTest@4803b726]" started-at="2018-01-10T14:10:44Z" name="DoLoginWithFacbook" finished-at="2018-01-10T14:10:52Z" duration-ms="7337" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DoLoginWithFacbook -->
      </class> <!-- Tests.SignUpByFacbookTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
